apps/web/src/pages/Teams/ViewTeamPage/ViewTeamPage.tsx

```diff
@@ -1,12 +1,78 @@
 import * as React from "react";
 import { useIntl } from "react-intl";
 import { BuildingOffice2Icon } from "@heroicons/react/24/outline";
-
+import { Scalars, useViewTeamQuery } from "~/api/generated";
+import { useParams } from "react-router-dom";
 import PageHeader from "@common/components/PageHeader";
 import SEO from "@common/components/SEO/SEO";
+import { commonMessages } from "@common/messages";
+import Pending from "@common/components/Pending";
+import NotFound from "@common/components/NotFound";
+import Link from "@common/components/Link";
+import useRoutes from "~/hooks/useRoutes";
+import Heading from "@common/components/Heading";
+import useLocale from "@common/hooks/useLocale";
+import { ViewTeam } from "./components/ViewTeam";
+
+type RouteParams = {
+  teamId: Scalars["ID"];
+};
+
+interface ViewTeamApiProps {
+  pageTitle: string;
+}
+
+export const ViewTeamApi = ({ pageTitle }: ViewTeamApiProps) => {
```

Comment:
Actually @esizer, this relates to the pattern we were discussing the other day, what do you think?

Reply:
So, I think this is what I am doing in https://github.com/GCTC-NTGC/gc-digital-talent/pull/5707

You can see how it works in https://github.com/GCTC-NTGC/gc-digital-talent/blob/03f8e8b13737ded63469c32bcc3f8a238cd35790/apps/web/src/pages/Pools/PoolPage/PoolPage.tsx

Reply:
It will be better if we implement loaders, as I mentioned in the details of the PR.

Reply:
is there anything I ought to implement from that without exploding the issue scope?

Reply:
@tristan-orourke might have a different opinion but I wouldn't worry about it. I planned on waiting until these team pages were setup and then work them into what is going on in #5707 👍

Reply:
Good point @esizer, this is going to get reworked in #5707 anyway, but I think it's worth socializing some of the patterns we want as early as possible.
Without blowing the scope, I'd suggest the following:
- move `useViewTeamQuery` into the ViewTeamPage component, and pass any variables needed to the nested component
- rename the nested component accordingly, from ViewTeamApi to ViewTeamContent
- Move the `<Pending>` component to the ViewTeamPage component, so that its in the same place as the query hook